MAUI/NavigationDrawer/Getting-Started.md

```diff
@@ -0,0 +1,332 @@
+
+### Adding the .NET MAUI NaviagtionDrawer control
+
+Step 1: Add the NuGet to the project as discussed in the above reference section. 
+
+Step 2: Add the namespace as shown in the following code sample:
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    xmlns:navigationdrawer="clr-namespace:Syncfusion.Maui.NavigationDrawer;assembly=Syncfusion.Maui.NavigationDrawer"
+	
+{% endhighlight %}
+
+{% highlight c# %}
+
+    using Syncfusion.Maui.NavigationDrawer;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+Step 3: Set the control to content in `ContentPage.`
+
+{% tabs %}
+
+{% highlight xaml %}
+
+
+<navigationdrawer:SfNavigationDrawer x:Name="navigationDrawer">
+    <navigationdrawer:SfNavigationDrawer.ContentView>
+        <Grid/>
+    </navigationdrawer:SfNavigationDrawer.ContentView>
+</navigationdrawer:SfNavigationDrawer>
+
+
+{% endhighlight %}
+
+{% highlight c# %}
+          
+SfNavigationDrawer navigationDrawer = new SfNavigationDrawer();
+Grid grid = new Grid();
+navigationDrawer.ContentView = grid;
+this.Content = navigationDrawer; 
+
+{% endhighlight %}
+
+{% endtabs %}
+
+N> It is mandatory to set [`ContentView`]() for [`SfNavigationDrawer`]() on initializing.
+
+## Adjust Drawer Size
+
+The default position of the navigation pane is on the left, so let's change the drawer width to 250.
+
+{% tabs %}	
+{% highlight xaml %}
+
+<navigationdrawer:SfNavigationDrawer x:Name="navigationDrawer">
+    <navigationdrawer:SfNavigationDrawer.DrawerSettings>
+        <navigationdrawer:DrawerSettings DrawerWidth="250"/>
+    </navigationdrawer:SfNavigationDrawer.DrawerSettings>
+    <navigationdrawer:SfNavigationDrawer.ContentView>
+        <Grid/>
+    </navigationdrawer:SfNavigationDrawer.ContentView>
+</navigationdrawer:SfNavigationDrawer>
+	
+{% endhighlight %}
+{% highlight c# %} 
+
+using Syncfusion.Maui.NavigationDrawer;
+
+namespace NavigationSample;
+
+public partial class NavigationDrawerPage : ContentPage
+{
+	public NavigationDrawerPage()
+	{
+		InitializeComponent();
+        SfNavigationDrawer navigationDrawer = new SfNavigationDrawer();
+        Grid grid = new Grid();
+        navigationDrawer.ContentView = grid;
+        navigationDrawer.DrawerSettings = new DrawerSettings()
+        {
+            DrawerWidth = 250,
+        };
+        this.Content = navigationDrawer;
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+
+N> To change the side of the navigation pane, use the [`Position`]() property. Use the [`DrawerHeight`]() property to change the drawer height in the Top and Bottom positions.
+
+## Add Hamburger Menu for Toggling Drawer
+
+Create a image button and set required image to the [`Image`](https://help.syncfusion.com/maui/button/customization#imagesource) property of Button. Subscribe Clicked event of the button and invoke ToggleDrawer() method in it to toggle the drawer. Properly align the layout of ['ContentView']() to position the hamburger icon at the top left, as shown in the following code:
```

Comment:
Create an ImageButton and set the required image to the 'Source' property. Subscribe to the Clicked event of the button and invoke the `ToggleDrawer()` method within it to toggle the drawer. Properly align the layout of `ContentView` to position the hamburger icon at the top left, as demonstrated in the following code,

Reply:
Changed